content/en/developers/metrics/_index.md

```diff
@@ -17,20 +18,28 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric (with [some exception][2]).
+
+A custom metric refers to **a unique combination of metric name, host, and tag values**. In general, any metric you send using [DogStatsD][3], or through a [custom Agent Check][4] is a custom metric. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][5].
+
+**Note**: Administrative users (those with [Datadog Admin roles][6] can see the total custom metrics per hour and the top 500 custom metrics for their account in the [usage details page][7]. Learn more about [how Custom metrics are counted][5].
+
+## Metric definition
```

Comment:
Why are we linking to what a metric is - isn't this documentation about custom metrics specifically?